xtask/src/ci.rs

```diff
@@ -185,6 +185,7 @@ fn run_crypto_tests() -> Result<()> {
     )
     .run()?;
     cmd!("rustup run stable cargo test -p matrix-sdk-crypto --features=backups_v1").run()?;
+    cmd!("rustup run stable cargo test -p matrix-crypto-ffi").run()?;
```

Comment:
not fond of just adding test after test here, as it means we can't parallelize them. if we end up adding more tests to this, we should consider switching to the same model as above and have CI fan them out...